mmv1/products/vertexai/api.yaml

```diff
@@ -96,13 +96,232 @@ objects:
+        output: true
+      - !ruby/object:Api::Type::String
+        name: displayName
+        description: Required. The display name of the Endpoint. The name can be up to 128 characters long and can consist of any UTF-8 characters.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: description
+        description: The description of the Endpoint.
+      - !ruby/object:Api::Type::Array
+        name: deployedModels
+        description: Output only. The models deployed in this Endpoint. To add or remove DeployedModels use EndpointService.DeployModel and EndpointService.UndeployModel respectively.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: deployedModels
+          description: Output only. The models deployed in this Endpoint. To add or remove DeployedModels use EndpointService.DeployModel and EndpointService.UndeployModel respectively.
+          properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: dedicatedResources
+                description: A description of resources that are dedicated to the DeployedModel, and that need a higher degree of manual configuration.
+                output: true
+                properties:
+                  - !ruby/object:Api::Type::NestedObject
+                    name: machineSpec
+                    description: The specification of a single machine used by the prediction.
+                    output: true
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: machineType
+                        description: 'The type of the machine. See the [list of machine types supported for prediction](https://cloud.google.com/vertex-ai/docs/predictions/configure-compute#machine-types) See the [list of machine types supported for custom training](https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types). For DeployedModel this field is optional, and the default value is `n1-standard-2`. For BatchPredictionJob or as part of WorkerPoolSpec this field is required. TODO(rsurowka): Try to better unify the required vs optional.'
+                        output: true
+                      - !ruby/object:Api::Type::Enum
+                        name: acceleratorType
+                        description: The type of accelerator(s) that may be attached to the machine as per accelerator_count.
+                        output: true
+                        values:
+                          - :ACCELERATOR_TYPE_UNSPECIFIED
+                          - :NVIDIA_TESLA_K80
+                          - :NVIDIA_TESLA_P100
+                          - :NVIDIA_TESLA_V100
+                          - :NVIDIA_TESLA_P4
+                          - :NVIDIA_TESLA_T4
+                          - :NVIDIA_TESLA_A100
+                          - :TPU_V2
+                          - :TPU_V3
```

Comment:
Lets change this to a string with a link to value types. 

Enum's are difficult to maintain

Reply:
What should the link to value types look like? Do you have an example?

Reply:
sorry missed this comment ! I see you got it sorted on your most recent change. Resolving this.